package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "cheerio": "^1.0.0-rc.2",
     "debug": "^2.2.0",
+    "fast-xml-parser": "^3.15.0",
```

Comment:
We already have xml2js for this purpose. If there's a good reason to switch libraries then that should happen on a separate PR and we should only keep one of them.